src/Features/CSharp/CSharpFeaturesResources.resx

```diff
@@ -228,6 +228,9 @@
   <data name="Deprecated" xml:space="preserve">
     <value>deprecated</value>
```

Comment:
I could argue about this, but only if you are open to discussion.

1. DA also has DeprecationType. Are you going to hide that? They are different.

2. True, but why dress that. Let's show the truth: [Obsolete], [Deprecated], [Deprecated(Remove)]

3. This is valuable, because custom attributes are valuable, e.g. for AOP: https://www.postsharp.net/model/code-contracts . This same as hiding or showing method parameters or visibility.

*"inches taller"* - let's be honest:

1) that is a super-specific case, I could come up with a method that would have an enormous signature and you would face the very same issue;

2) heavily-overloaded methods are the same issue, yet it is handled properly;

3) can be turned off *by default*, AOP-driven developers and people like me will enable it - no problem.


I understand that your team has a lot of requests to deal with, but postponing would be better than dismissing.